src/tenable/test/test_export_assets.py

```diff
@@ -0,0 +1,84 @@
+import pytest
+from datetime import datetime
+from dataclasses import dataclass
+
+from src.tenable.credentials import TenableCredentials
+from src.tenable.export_assets import export_tenable_assets
+
+@dataclass
+class MockResponse:
+    status_code: int
+    json_data: dict
+    text: str
+
+    def json(self):
+        return self.json_data
+
+@dataclass
+class MockThread:
+    def start():
+        pass
+
+    def join():
+        pass
+
+@pytest.fixture(autouse=True)
+def mock_time():
+    now = datetime.today()
+    return now.timestamp()
+
+@pytest.fixture(autouse=True)
+def cred_object():
+    return TenableCredentials(
+        access_key="ACCESS_KEY", 
+        secret_key="SECRET_KEY"
+    )
+
+@pytest.fixture(autouse=True)
+def post_response_success():
+    return MockResponse(
+        status_code=200, 
+        json_data={"export_uuid": "EXPORT_UUID"}, 
+        text="TEXT"
+    )
+
+@pytest.fixture(autouse=True)
+def post_response_failure():
+    return MockResponse(
+        status_code=400, 
+        json_data={"export_uuid": "EXPORT_UUID"}, 
+        text="TEXT"
+    )
+
+@pytest.fixture(autouse=True)
+def get_response():
+    return MockResponse(
+        status_code=200, 
+        json_data={
+            "status": "FINISHED", 
+            "chunks_available": [1]
+        }, 
+        text="TEXT"
+    )
+
+@pytest.fixture(autouse=True)
+def mock_thread():
+    return MockThread()
+
+def test_export_tenable_assets_success(mocker, cred_object, post_response_success, get_response, mock_thread, mock_time):
+    mocker.patch("requests.post", return_value=post_response_success) 
+    mocker.patch("requests.get", return_value=get_response)
```

Comment:
instead of mocking `requests` functions, consider using [`requests-mock`](https://pypi.org/project/requests-mock/) instead, which lets you map urls to mocked responses. it is used in #4, so you can use it as a reference :)

Reply:
Implemented `requests-mock` in tests that interact require responses